buildSrc/src/main/kotlin/io/spine/internal/gradle/DependencyResolution.kt

```diff
@@ -59,7 +62,7 @@ fun doForceVersions(configurations: ConfigurationContainer) {
 /**
  * Forces dependencies used in the project.
  */
-fun ConfigurationContainer.forceVersions() {
```

Comment:
I would leave the previous version of the API as well, but mark it as deprecated. This will make the migration much easier.